api/conf/settings/thumbnails.py

```diff
@@ -16,18 +16,10 @@
 THUMBNAIL_WIDTH_PX = config("THUMBNAIL_WIDTH_PX", default="600")
 THUMBNAIL_QUALITY = config("THUMBNAIL_JPG_QUALITY", default="80")
 
-THUMBNAIL_TIMEOUT_PREFIX = config(
-    "THUMBNAIL_TIMEOUT_PREFIX", default="thumbnail_timeout:"
+THUMBNAIL_ERROR_INITIAL_ALERT_THRESHOLD = config(
+    "THUMBNAIL_ERROR_INITIAL_ALERT_THRESHOLD", default=100, cast=int
 )
 
-THUMBNAIL_HTTP_ERROR_PREFIX = config(
-    "THUMBNAIL_HTTP_ERROR_PREFIX", default="thumbnail_http_error:"
-)
-
-THUMBNAIL_HTTP_ERROR_THRESHOLD_TO_NOTIFY = config(
-    "THUMBNAIL_HTTP_ERROR_THRESHOLD_TO_NOTIFY", default=1000, cast=int
-)
-
-THUMBNAIL_HTTP_ERROR_THRESHOLD_FOR_REPEATED_NOTIFICATIONS = config(
-    "THUMBNAIL_HTTP_ERROR_THRESHOLD_FOR_REPEATED_NOTIFICATIONS", default=1000, cast=int
+THUMBNAIL_ERROR_REPEATED_ALERT_THRESHOLD = config(
```

Comment:
Nit: `_THRESHOLD` is not the right name for this since it is more of a frequency/interval.

Reply:
True, I'll adjust!